database/rrdhost.c

```diff
@@ -707,14 +707,31 @@ void rrdhost_save_charts(RRDHOST *host) {
     rrdhost_unlock(host);
 }
 
+static int is_valid_label(char *key) {
+    while(key) {
+        if(!isdigit(*key) || !isalpha(*key) || *key != '.' || *key != '_' || *key != '-')
```

Comment:
Function name is `is_valid_label`, do we need to check label value. Or valid key == valid label. Or this is `is_valid_label_key`?

Reply:
No it is only a check for the key, I think it is better to allow the users define the labels that they prefer, for keys would be more complicated, this is the motive we are defining this function.

Reply:
I accepted your suggestion and I create a new commit to rename the function.

Reply:
I see label value has req

#7410

> Label values can be UTF-8 strings.

Reply:
ok, fixed 😄